usaspending_api/references/models.py

```diff
@@ -482,8 +482,8 @@ class Meta:
         unique_together = (('program_activity_code', 'budget_year', 'responsible_agency_id', 'allocation_transfer_agency_id', 'main_account_code'),)
 
 
-class CFDA(DataSourceTrackedModel):
-    program_number = models.TextField(null=False)
+class Cfda(DataSourceTrackedModel):
+    program_number = models.TextField(null=False, db_index=True)
```

Comment:
You're right, I meant to put a unique on this (which comes with an index for free) instead of vice versa, let me fix that.